doc/langref.html.in

```diff
@@ -10261,7 +10267,7 @@ pub fn main() void {
     foo(Set1.B);
 }
 fn foo(set1: Set1) void {
-    const x = @errSetCast(Set2, set1);
+    const x = @as(Set2, @errSetCast(set1));
```

Comment:
This line could be written `const x: Set2 = @errSetCast(set1);`, or changed to discard the result like in the test case at `comptime` above.

Reply:
Ah yes - I'd like to see how CI does before pushing any changes, but I'll make a note of this one, thanks.

Reply:
Resolved in follow-up pr #16192